### PR TITLE
Install docker-compose packages.

### PR DESCRIPTION
**Pull request by bendory**
Fixes https://github.com/GoogleCloudPlatform/cloud-builders/issues/835

### CODE REVIEW

File: docker/Dockerfile-base [hunk 1]
```diff
@@ -8,4 +8,7 @@ RUN apt-get -y update && \
         make \
         software-properties-common && \
     curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
-    apt-key fingerprint 0EBFCD88
+    apt-key fingerprint 0EBFCD88 && \
+    add-apt-repository \
+       "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
+	apt-get clean
```
Comment (leeonlee) [inline docker/Dockerfile-base:14]: Dedent

File: docker/Dockerfile-versioned [hunk 2]
```diff
@@ -2,9 +2,8 @@ FROM temp
 
 ARG DOCKER_VERSION=VERSION
 
-RUN add-apt-repository \
-       "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
-    apt-get -y update && \
-    apt-get -y install docker-ce=${DOCKER_VERSION} docker-ce-cli=${DOCKER_VERSION}
+RUN apt-get -y install docker-ce=${DOCKER_VERSION} docker-ce-cli=${DOCKER_VERSION} \
+	    docker-compose docker-compose-plugin &&\
+	apt-get clean
```
Comment (leeonlee) [inline docker/Dockerfile-versioned:7]: Also dedent

Comment (leeonlee) [inline docker/Dockerfile-versioned:7]: Also dedent

File: docker/Dockerfile-versioned [hunk 3]
```diff
@@ -2,9 +2,8 @@ FROM temp
 
 ARG DOCKER_VERSION=VERSION
 
-RUN add-apt-repository \
-       "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
-    apt-get -y update && \
```
Comment (leeonlee) [inline docker/Dockerfile-versioned:7]: Do we need to run `apt-get update` after `add-apt-repository`? If so, we need this line

Comment (leeonlee) [inline docker/Dockerfile-versioned:7]: Do we need to run `apt-get update` after `add-apt-repository`? If so, we need this line

**Review by leeonlee — CHANGES_REQUESTED** [review]

File: docker/Dockerfile-versioned (hunk 2 above)
Comment (bendory) [reply docker/Dockerfile-versioned:7]: tab weirdness; cleaned up

File: docker/Dockerfile-versioned (hunk 3 above)
Comment (bendory) [reply docker/Dockerfile-versioned:7]: apparently not -- everything works without it (which admittedly surprised me after adding the repository)

**Pull request merged**